src/ngx_http_lua_timer.c

```diff
@@ -25,7 +25,8 @@ typedef struct {
      * ngx_connection_t. and we use the Lua coroutine reference number as
      * the event ident */
     int           co_ref;
-    unsigned      premature;  /* :1 */
+    unsigned      delay:31;
+    unsigned      premature:1;
```

Comment:
Better move these smaller fields to the end of the struct. The subsequent pointer fields are all bigger than these :)